### DISCUSSION TITLE
How to add multiple tags to a work item?

### DISCUSSION THREAD

**Discussion by annaowens** [Microsoft/mail2bug#125] [Q&A] [answered]
Hi all 👋 

I would like to add multiple tags to a work item created by a support thread. Currently, my team sets a default tag that is applied to each work item created:
`<DefaultValueDefinition Field="Tags" Value="Support" />`

What is the syntax to add 2+ tags by default? 

Ex. Something like `Value="Support, 2022"` to generate two tags, "Support" and "2022"?

**Comment by davisjr** [comment, marked as answer]
That should just work - maybe skip the space?

`Value="Support,2022"`
